crates/ruff_python_parser/src/lexer.rs

```diff
@@ -407,6 +407,22 @@ impl<'source> Lexer<'source> {
         #[cfg(debug_assertions)]
         debug_assert_eq!(self.cursor.previous(), '#');
 
+        #[cfg(any(
+            target_arch = "x86_64",
+            target_arch = "aarch64",
+            target_arch = "wasm32"
+        ))]
```

Comment:
I think `memchr` handl this automatically and not having the configuration ensures that newly supported platforms automatically profit from a faster `memchr` routine.

Reply:
True, will look into this more.